Template.php

```diff
@@ -1693,23 +1696,7 @@ protected function tidy() {
       else $this->set('others', $others);
     }
 
-    if ($this->added('journal') || $journal && $this->added('issn')) $this->forget('issn');
-
-    if ($journal) {
```

Comment:
Perhaps the line setting it has been removed: $journal would originally have been set to $this->get('journal'), possibly falling back to something like $this->get('periodical') if 'journal' was not set.